src/order/atoms.lean

```diff
@@ -96,4 +185,43 @@ theorem is_simple_lattice_iff_is_atom_top [bounded_lattice α] :
 
 theorem is_simple_lattice_iff_is_coatom_bot [bounded_lattice α] :
   is_simple_lattice α ↔ is_coatom (⊥ : α) :=
-iff.trans is_simple_lattice_iff_is_simple_lattice_order_dual is_simple_lattice_iff_is_atom_top
+is_simple_lattice_iff_is_simple_lattice_order_dual.trans is_simple_lattice_iff_is_atom_top
+
+namespace set
+
+theorem is_simple_lattice_Iic_iff_is_atom [bounded_lattice α] {a : α} :
+  is_simple_lattice (Iic a) ↔ is_atom a :=
+is_simple_lattice_iff_is_atom_top.trans $ and_congr (not_congr subtype.mk_eq_mk)
+  ⟨λ h b ab, subtype.mk_eq_mk.1 (h ⟨b, le_of_lt ab⟩ ab),
+    λ h ⟨b, hab⟩ hbotb, subtype.mk_eq_mk.2 (h b (subtype.mk_lt_mk.1 hbotb))⟩
+
+theorem is_simple_lattice_Ici_iff_is_coatom [bounded_lattice α] {a : α} :
+  is_simple_lattice (Ici a) ↔ is_coatom a :=
+is_simple_lattice_iff_is_coatom_bot.trans $ and_congr (not_congr subtype.mk_eq_mk)
+  ⟨λ h b ab, subtype.mk_eq_mk.1 (h ⟨b, le_of_lt ab⟩ ab),
+    λ h ⟨b, hab⟩ hbotb, subtype.mk_eq_mk.2 (h b (subtype.mk_lt_mk.1 hbotb))⟩
+
+end set
+
+namespace order_iso
+
+variables [bounded_lattice α] {β : Type*} [bounded_lattice β] (f : α ≃o β)
+include f
+
+lemma is_atom_iff (a : α) : is_atom a ↔ is_atom (f a) :=
+and_congr (not_congr ⟨λ h, f.map_bot ▸ (congr rfl h), λ h, (f.injective (f.map_bot.symm ▸ h))⟩)
+  ⟨λ h b hb, f.symm.injective begin
+    rw f.symm.map_bot,
+    apply h,
+    rw [← f.symm_apply_apply a],
+    exact (f.symm : β ↪o α).map_lt_iff.1 hb,
+  end,
+  λ h b hb, f.injective ((h (f b) ((f : α ↪o β).map_lt_iff.1 hb)).trans f.map_bot.symm)⟩
+
+lemma is_coatom_iff (a : α) : is_coatom a ↔ is_coatom (f a) := f.dual.is_atom_iff a
+
+lemma is_simple_lattice_iff (f : α ≃o β) : is_simple_lattice α ↔ is_simple_lattice β :=
+by rw [is_simple_lattice_iff_is_atom_top, is_simple_lattice_iff_is_atom_top,
+  f.is_atom_iff ⊤, f.map_top]
```

Comment:
Please add `order_iso.is_simple_lattice` as an alias for `iff.mpr` of this `iff`.